src/styles/partials/components/_accordion.scss

```diff
@@ -0,0 +1,74 @@
+.dg_accordion__collapsible {
+  color: $blue;
+  font-family: $default-heading-font;
+  font-size: $h2-font-size;
+  font-weight: $font-bold;
+  width: 100%;
+  border-bottom: 1px $black solid;
```

Comment:
Recommend changing 1px to use the sass variable: $border-width-small.

Reply:
Fixed